gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -292,6 +286,12 @@ private static void configureErrorProneOptions(
         }
     }
 
+    static String excludedPathsRegex() {
+        // don't want backslashes on windows to break our regex
+        String separator = File.separator.contains("\\") ? Pattern.quote("\\") : File.separator;
+        return String.format(".*%s(build|generated_.*[sS]rc|src%sgenerated.*)%s.*", separator, separator, separator);
```

Comment:
@iamdanfox we might be able to remove `build` from regex here, or might want to constrain the pattern to avoid falsely matching say `com/foo/build/Bar.java`.

Reply:
Yeah this feels a bit dangerous. I'd suggest we do something like `build.*generated` since almost all code-gen plugins I've seen put source files on a path with a `generated` component.